src/TextSerializationWriter.cs

```diff
@@ -23,7 +23,7 @@ public TextSerializationWriter()
         writer = new(
             _stream,
             // Default encoding
-            encoding: System.Text.Encoding.UTF8,
+            encoding: new System.Text.UTF8Encoding(false, true),
```

Comment:
```suggestion
```

Reply:
It doesn't build that way. I think netstandard2.0 and netstandard2.1 don't support named params?

```
C:\Source\Repos\kiota-serialization-text-dotnet\src\TextSerializationWriter.cs(23,18): error CS7036: There is no argument given that corresponds to the required parameter 'encoding' of 'StreamWr
iter.StreamWriter(Stream, Encoding, int, bool)' [C:\Source\Repos\kiota-serialization-text-dotnet\src\Microsoft.Kiota.Serialization.Text.csproj::TargetFramework=netstandard2.0]
C:\Source\Repos\kiota-serialization-text-dotnet\src\TextSerializationWriter.cs(23,18): error CS7036: There is no argument given that corresponds to the required parameter 'encoding' of 'StreamWr
iter.StreamWriter(Stream, Encoding, int, bool)' [C:\Source\Repos\kiota-serialization-text-dotnet\src\Microsoft.Kiota.Serialization.Text.csproj::TargetFramework=netstandard2.1]
```

Reply:
thanks for double checking